app/javascript/react/components/Map/Markers/SingleMarker/SingleMarker.style.tsx

```diff
@@ -53,16 +53,16 @@ const DataContainer = styled.div`
 `;
 
 const MarkerCircle = styled.div<MarkerProps>`
-  width: 2rem;
-  height: 2rem;
+  width: 1.2rem;
+  height: 1.2rem;
   border-radius: 50%;
   background-color: ${(props) => props.color};
   pointer-events: none;
 `;
 
 const MarkerText = styled(H4)`
   color: ${gray400};
-  font-size: 1.2rem;
+  font-size: 12px;
```

Comment:
Is it different size from 1.2rem?

Reply:
According to chatgpt yes 🤔

Reply:
Should I change it?

Reply:
If it looks the same i would keep 1.2rem if not 12px